### PR TITLE
Update select-domain-to-use-for-email-from-microsoft-365-products.md

### LINKED ISSUE #12187: Slight correction on verbiage
The references to the 'Organization profile' are "Organizational Settings profile" and "On the Org settings page, select Organizational profile.
On the Organizational profile page,"

This should be 'Organization profile', not 'Organizational profile' or 'Organizational Settings profile'.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: eda44995-3561-2f08-e776-5359b0aaf37c
* Version Independent ID: eda44995-3561-2f08-e776-5359b0aaf37c
* Content: [Select the domain to use for email from Microsoft 365 products - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/email/select-domain-to-use-for-email-from-microsoft-365-products?view=o365-worldwide)
* Content Source: [microsoft-365/admin/email/select-domain-to-use-for-email-from-microsoft-365-products.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/email/select-domain-to-use-for-email-from-microsoft-365-products.md)
* Service: **microsoft-365-business**
* GitHub Login: @efrene
* Microsoft Alias: **efrene**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12187

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/email/select-domain-to-use-for-email-from-microsoft-365-products.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The default option is that users receive no-reply emails sent by Microsoft on be
 
 ## <a name="configsetting">Configure the "Send email notifications from your domain" setting</a>
 
-The setting is available via the Microsoft 365 Admin Center Settings menu item. Select Settings, select Org Settings, and then the Organizational Settings profile tab as illustrated below. 
+The setting is available via the Microsoft 365 Admin Center Settings menu item. Select Settings, select Org Settings, and then the Organization profile tab as illustrated below.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/select-domain-to-use-for-email-from-microsoft-365-products.md]:
```suggestion
The setting is available via the Microsoft 365 Admin Center Settings menu item. Select Settings, select Org settings, and then select the Organization profile tab as illustrated below. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: microsoft-365/admin/email/select-domain-to-use-for-email-from-microsoft-365-products.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ The default option is that users receive no-reply emails sent by Microsoft on be
 
 ## <a name="configsetting">Configure the "Send email notifications from your domain" setting</a>
 
-The setting is available via the Microsoft 365 Admin Center Settings menu item. Select Settings, select Org Settings, and then the Organizational Settings profile tab as illustrated below. 
+The setting is available via the Microsoft 365 Admin Center Settings menu item. Select Settings, select Org settings, and then select the Organization profile tab as illustrated below.
```
Comment (tiburd) [inline microsoft-365/admin/email/select-domain-to-use-for-email-from-microsoft-365-products.md]:
```suggestion
The setting is available via the Microsoft 365 Admin Center **Settings** menu item. Select **Settings**, select **Org settings**, and then select the **Organization profile** tab as illustrated below. 
```

**Pull request merged**